bitnami/cassandra/values.yaml

```diff
@@ -0,0 +1,174 @@
+## Global Docker image registry
+## Please, note that this will override the image registry for all the images, including dependencies, configured to use the global value
+##
+# global:
+#   imageRegistry:
+
+## Bitnami Cassandra image version
+## ref: https://hub.docker.com/r/bitnami/cassandra/tags/
+##
+image:
+  registry: docker.io
+  repository: bitnami/cassandra
+  ## Bitnami Cassandra image tag
+  ## ref: https://github.com/bitnami/bitnami-docker-cassandra#supported-tags-and-respective-dockerfile-links
+  ##
+  tag: 3.11.3
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: Always
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  # pullSecrets:
+  #   - myRegistrKeySecretName
+
+## Specify a service type
+## ref: http://kubernetes.io/docs/user-guide/services/
```

Comment:
We usually add a line with `##` between the description and the code, something like:
```
## Bitnami Cassandra image version
## ref: https://hub.docker.com/r/bitnami/cassandra/tags/
##
image:
  registry: docker.io
  repository: bitnami/cassandra
```

Can you check other cases?